backend/tetra/api/resources.py

```diff
@@ -157,3 +157,13 @@ class ResultResource(Resource):
     ROUTE = "/projects/{project_id}/builds/{build_id}/results/{result_id}"
     RESOURCE_CLASS = Result
     RESOURCE_ID_KEY = "result_id"
+
+class SpuriousResource(Resource):
+    ROUTE = "/spurious/{since_time}"
```

Comment:
Then we accept the parameters:

* `type`: `none (default -> all)` | `nightly` (which is really just a string-prefix for the `SQL` filter).
* `since_time`

Reply:
yeah, taking the parameter approach would be my biggest gripe there... since the `since_time` is IMO not an identifying aspect of the resource in any way, but rather a... parameter 🙈...
even the `failures` could be a `status` query or something along those lines...
if the `/tests` might ever be used to reference a single test I would rather put the statistics on a higher level and do:`/test-statistics` to allow `/tests/{test_id}` and not interfere with our famous test called `statistics` 🙄... but that depends a bit on where we want to go with the API/ page I guess...

as to how to design a good REST API - I'm afraid I can't put this in a comment here (for I'm not a pro at that either + there are way better people in the world to learn from e.g. other APIs that have done this for a while & many consumers such as [twitter](https://developer.twitter.com/en/docs/twitter-api/tweets/timelines/api-reference/get-users-id-tweets))

Reply:
haha, the `since_time` was never supposed to be in the `URL`, I just wanted to get this to work, and then do a quick discussion on the API here.

for now I did go with: `/api/tests/statistics/spurious-failures/`, but feel free to intervene here. Parameters are passed in the URL, and can be `test_name`, `type`, `since_time`